src/utils/parameters.py

```diff
@@ -48,6 +51,12 @@ def arg_group_base(parser):
         help='Build number of the game files to be used. Defaults to current build',
         default=os.getenv('BUILD_NUM', None),
     )
+    group_base.add_argument(
+        '-v',
+        '--verbose',
```

Comment:
i get an error when using `--verbose`

Reply:
It seems I'm confused on how this works; I can't find any other instances of "build_num" or "BUILD_NUM" in the repo that cause it to work where "VERBOSE" doesn't. Do you know?

Reply:
build num is a bad example to use, as that takes in an argument. You want one that doesn't, so take a look at some other existing options for comparison